README.md

```diff
@@ -21,23 +21,25 @@ When opening a PR, please make sure to follow our [contribution guide](CONTRIBUT
 
 The Confluent CLI is available to install for macOS, Linux, and Windows.
 
+#### Homebrew
+
+Install the latest version of `confluent` to `/usr/local/bin`:
+
+    brew install confluentinc/tap/cli
+
 #### Scripted installation
 
 Install the latest version of `confluent` to `/usr/local/bin`:
 
     curl -sL https://raw.githubusercontent.com/confluentinc/cli/main/install.sh | sh -s -- -b /usr/local/bin
 
-#### macOS
+Print a complete list of versions available for download:
 
-1. Download the latest macOS tar.gz file for your architecture type from https://github.com/confluentinc/cli/releases/latest
-2. Unzip the file: `tar -xvf confluent_X.X.X_darwin_XXXXX.tar.gz`
-3. Move `confluent` to a folder in your `$PATH`, such as `/usr/local/bin`
+    curl -sL https://raw.githubusercontent.com/confluentinc/cli/main/install.sh | sh -s -- -l
 
-#### Linux
+Install `confluent` v3.6.0 to `/usr/local/bin`:
 
-1. Download the latest Linux tar.gz file for your architecture type from https://github.com/confluentinc/cli/releases/latest
-2. Unzip the file: `tar -xvf confluent_X.X.X_linux_XXXXX.tar.gz`
-3. Move `confluent` to a folder in your `$PATH`
+    curl -sL https://raw.githubusercontent.com/confluentinc/cli/main/install.sh | sh -s -- -b /usr/local/bin 3.6.0
```

Comment:
what is `raw.githubusercontent.com`? is it stable / public / can trust?

Reply:
Yes to all three! It's owned by GitHub.